src/main/java/build/buildfarm/worker/cgroup/Group.java

```diff
@@ -23,21 +23,18 @@
 import java.nio.file.Paths;
 import java.util.List;
 import javax.annotation.Nullable;
+import lombok.Getter;
 import lombok.extern.java.Log;
 
 @Log
 public final class Group {
-  private static final Group root = new Group(/* name= */ null, /* parent= */ null);
+  @Getter private static final Group root = new Group(/* name= */ null, /* parent= */ null);
   private static final Path rootPath = Paths.get("/sys/fs/cgroup");
 
   private @Nullable final String name;
```

Comment:
this could use on too

Reply:
@werkt what did you mean by this comment?